### PR TITLE
Simplify download file properties

### PR DESCRIPTION
**Pull request by DanVanAtta**
Simplifies the '*.properties' files that we create after downloading a map zip. Largely we store extraneous data in them like the engine version and time of download, we only use the map version from those files.

commit 78927f16a0fa42c70367b03850102854af48a954

    Remove unused properties in map '*.properties' files, we only use the map version
    
    The map url, time of download, engine version are unused, we only
    need the map version in a '.properties' map suffix file.

commit 0c39898c5d89b72616da85cfb8c74ccde9d08517

    Simplify DownloadFileProperties construction
    
    Rather than construct via no-arg and then inject
    a version value, we can accept the version value
    as a constructor argument.

### CODE REVIEW

**Comment by DanVanAtta** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ game-core/src/main/java/games/strategy/engine/framework/map/download/FileSystemAccessStrategy.java  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/triplea-game/triplea/pullRequest?prid=6109245&bid=19995526)

File: game-core/src/main/java/games/strategy/engine/framework/map/download/DownloadFileProperties.java [hunk 1]
```diff
@@ -35,9 +38,9 @@ static DownloadFileProperties loadForZip(final File zipFile) {
     return downloadFileProperties;
   }
 
-  static void saveForZip(final File zipFile, final DownloadFileProperties props) {
+  void saveForZip(final File zipFile) {
     try (OutputStream fos = new FileOutputStream(fromZip(zipFile))) {
-      props.props.store(fos, null);
```
Comment (RoiEXLab) [inline game-core/src/main/java/games/strategy/engine/framework/map/download/DownloadFileProperties.java:40]: 😮

**Review by RoiEXLab — APPROVED** [review]

**Comment by RoiEXLab** [comment]
The build is failing, I'm assuming the same problem we have in #7505

**Comment by DanVanAtta** [comment]
Tends to always be the branch build and not the PR build.
The error's are seemingly two classes, largely the two things we do around sockets for game listings and chat.
Does seem there is a race condition that is exposed now.

```
GameListingWebsocketIntegrationTest > Post a game, verify listener is notified FAILED

    Wanted but not invoked:

    gameUpdatedListener.accept(

        LobbyGameListing(gameId=118d5000-4a70-44f1-b67a-8a92222adaae, lobbyGame=LobbyGame(hostAddress=127.0.0.1, hostPort=12, hostName=name, mapName=map, playerCount=3, gameRound=1, epochMilliTimeStarted=1599030577598, mapVersion=1, passworded=false, status=Waiting For Players, comments=comments))

    );

    -> at org.triplea.modules.game.GameListingWebsocketIntegrationTest.verifyPostGame(GameListingWebsocketIntegrationTest.java:94)

    Actually, there were zero interactions with this mock.
```

**Pull request merged**